README.md

```diff
@@ -163,15 +166,15 @@ If a published post on openliberty.io/blog contains an error or needs updating i
 
 3. If the blog tags need correcting, update the [blogs_tags.json](./blog_tags.json) file. If you add new tags, make sure to add the blog post's slug to the beginning of the `posts` arrays (1-2 entries per line) for each tag.
 
-4. Create a PR from your branch to the `draft` branch, then (when the PR has been merged) run the [draft site build from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) to check that the changes are fine on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+4. Create a PR from your branch to the `draft` branch, then (when the PR has been merged) wait for IBM Cloud to rebuild [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/) (if you have access to [this Slack channel](https://app.slack.com/client/T15GKHBT4/C01GXGW1DGQ), then you can use it to track build/deploy preogress.  Check that the changes are fine on the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
 
-   Make any changes in your branch then push to `draft` branch again rebuild.
+   Make any changes in your branch then push to `draft` branch again verify the changes after rebuild.
```

Comment:
Make any changes in your branch then push to `draft` branch again verify the changes after rebuild.
-->
Make any changes in your branch, then push to the `draft` branch again  and verify the changes after rebuild.